### PR TITLE
Add package analyzer to the description

### LINKED ISSUE #260: Missing description of package analyzer
**Describe the bug**
`A job for scheduling solver to resolve dependency graphs of new packages` is the description of graph-regresh job but this component schedule also package analyzer and it is not described here.

**Expected behavior**
A review of README and description of this component adding information about package-analyzer is required.

### PR DESCRIPTION
**Pull request by bissenbay**
Fixes https://github.com/thoth-station/graph-refresh-job/issues/260

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,7 +1,8 @@
 thoth-graph-refresh-job
 -----------------------
 
-A job for scheduling solver to resolve dependency graphs of new packages
+A job for scheduling solver to resolve dependency graphs of new packages and package analyzer for packages that are not
```
Comment (pacospace) [inline README.rst:4]: WDYT about `A job for scheduling solver to resolve dependency graphs and package-analyzer to gather digests of new packages or packages not analyzer yet`

File: README.rst [hunk 2]
```diff
@@ -30,7 +31,7 @@ in Thoth's middletier namespace.
 
   * graph-refresh job is run in Thoth's frontend namespace
   * templates for solvers are picked from Thoth's infra namespace - see installed solvers endpoint on Management API for available solvers in the deployment
```
Comment (pacospace) [inline README.rst:33]: Are there templates also for package-analyzer?

File: README.rst (hunk 1 above)
Comment (bissenbay) [reply README.rst:4]: LGTM, thanks!

File: README.rst (hunk 2 above)
Comment (bissenbay) [reply README.rst:33]: If this https://github.com/thoth-station/package-analyzer/blob/master/openshift/job-template.yaml what you mean by templates, then I think yes.

Comment (pacospace) [reply README.rst:33]: Then we should modify also the line related to templates WDYT? :)

**Review by pacospace — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (fridex) [reply README.rst:4]: Not just digests, but also ABI (and we would like to add more over time).

**Pull request merged**